examples/lookup-based_wake_steering_florisstandin/compare_yaw_offset_designs.py

```diff
@@ -0,0 +1,255 @@
+        "enable_active_wake_mixing": False,
+        "wake_deflection_parameters": {
+            "gauss": {
+                "ad": 0.0,
+                "alpha": 0.58,
+                "bd": 0.0,
+                "beta": 0.077,
+                "dm": 1.0,
+                "ka": 0.38,
+                "kb": 0.004,
+            },
+        },
+        "wake_turbulence_parameters": {
+            "crespo_hernandez": {"initial": 0.1, "constant": 0.5, "ai": 0.8, "downstream": -0.32}
+        },
+        "wake_velocity_parameters": {
+            "gauss": {"alpha": 0.58, "beta": 0.077, "ka": 0.38, "kb": 0.004},
+        },
+    },
+    "farm": {
+        "layout_x": [0.0, 1000.0],
+        "layout_y": [0.0, 0.0],
+        "turbine_type": ["nrel_5MW"],
+    },
+    "flow_field": {
+        "wind_speeds": [8.0],
+        "wind_directions": [270.0],
+        "turbulence_intensities": [0.06],
+        "wind_veer": 0.0,
+        "wind_shear": 0.12,
+        "air_density": 1.225,
+        "reference_wind_height": 90.0,
+    },
+    "name": "GCH_for_FlorisStandin",
+    "description": "FLORIS Gauss Curl Hybrid model standing in for AMR-Wind",
+    "floris_version": "v4.x",
+}
+
+if __name__ == "__main__":
+    # Specify various settings for the lookup table designs
+    wd_resolution = 3.0
+    ws_resolution = 1.0
+    ws_min = 2.0
+    ws_max = 17.0
+    ti_resolution = 0.02
+    ti_min = 0.06
+    ti_max = 0.08
+    minimum_yaw_angle = -25.0
+    maximum_yaw_angle = 25.0
+    wd_std = 3.0
+    ws_main = 8.0
+    wd_rate_limit = 3.0
+    ws_rate_limit = 100.0 # No rate limit on wind speed
+    ti_rate_limit = 1e3 # No rate limit on turbulence intensity
+    plot_turbine = 0
+    plot_wd_lims = (240, 300)
+
+    # Plotting
+    col_simple = "black"
+    col_unc = "C0"
+    col_rate_limited = "C1"
+    col_ws_ramps = "C2"
+    
+    fmodel = FlorisModel(floris_dict)
+
+    print("Building simple lookup table.")
+    df_opt_simple = wsd.build_simple_wake_steering_lookup_table(
+        fmodel,
+        wd_resolution=wd_resolution,
+        ws_resolution=ws_resolution,
+        ws_min=ws_min,
+        ws_max=ws_max,
+        ti_resolution=ti_resolution,
+        ti_min=ti_min,
+        ti_max=ti_max,
+        minimum_yaw_angle=minimum_yaw_angle,
+        maximum_yaw_angle=maximum_yaw_angle,
+    )
+
+    print("\nBuilding lookup table with 3 degrees of wind direction uncertainty.")
+    df_opt_unc = wsd.build_uncertain_wake_steering_lookup_table(
+        fmodel,
+        wd_std=wd_std,
+        wd_resolution=wd_resolution,
+        ws_resolution=ws_resolution,
+        ws_min=ws_min,
+        ws_max=ws_max,
+        ti_resolution=ti_resolution,
+        ti_min=ti_min,
+        ti_max=ti_max,
+        minimum_yaw_angle=minimum_yaw_angle,
+        maximum_yaw_angle=maximum_yaw_angle,
+    )
+
+    print("\nApplying rate limits to simple lookup table (3 deg/deg).")
+    df_opt_rate_limited = wsd.apply_static_rate_limits(
```

Comment:
I'm ok leaving this in, but I think in general we've found this generally not too preferable approach versus rate limiting/filtering in the time domain.  Maybe we just add some comments to the effect that this was an idea but no longer preferred